requirements.in

```diff
@@ -12,6 +12,7 @@ yamllint
 
 # Build-Time (Not Part of pre-commit)
 build
+cdktf-cdktf-provider-aws
```

Comment:
More of an aside, but I have 100% moved to `uv` workspaces and it's great

Reply:
Interesting, thanks for the tip